17:00:23.828 Running build in Washington, D.C., USA (East) – iad1
17:00:23.828 Build machine configuration: 2 cores, 8 GB
17:00:23.989 Cloning github.com/Shriii19/FlowForge (Branch: enhancement/global-notifications-center, Commit: 2d99b57)
17:00:24.576 Cloning completed: 587.000ms
17:00:24.749 Found .vercelignore
17:00:24.752 Removed 0 ignored files defined in .vercelignore
17:00:25.395 Restored build cache from previous deployment (5DbcxoScn7aEoFcwdotLxXsTcLZY)
17:00:25.666 Running "vercel build"
17:00:25.714 Vercel CLI 54.4.1
17:00:25.975 Running "install" command: `npm install --no-workspaces`...
17:00:34.921 
17:00:34.922 added 15 packages, and audited 451 packages in 9s
17:00:34.923 
17:00:34.923 153 packages are looking for funding
17:00:34.924   run `npm fund` for details
17:00:34.959 
17:00:34.960 5 moderate severity vulnerabilities
17:00:34.960 
17:00:34.961 To address issues that do not require attention, run:
17:00:34.962   npm audit fix
17:00:34.962 
17:00:34.962 To address all issues (including breaking changes), run:
17:00:34.963   npm audit fix --force
17:00:34.963 
17:00:34.965 Run `npm audit` for details.
17:00:35.017 Detected Next.js version: 16.2.6
17:00:35.018 Running "npm run build"
17:00:35.154 npm warn config ignoring workspace config at /vercel/path0/frontend/.npmrc
17:00:35.183 
17:00:35.183 > frontend@0.1.0 build
17:00:35.184 > cross-env NODE_OPTIONS=--max-old-space-size=4096 next build
17:00:35.184 
17:00:36.645   Applying modifyConfig from Vercel
17:00:36.666 ▲ Next.js 16.2.6 (Turbopack)
17:00:36.667 
17:00:36.671 ⚠ The "middleware" file convention is deprecated. Please use "proxy" instead. Learn more: https://nextjs.org/docs/messages/middleware-to-proxy
17:00:36.709   Creating an optimized production build ...
17:00:53.342 Turbopack build encountered 1 warnings:
17:00:53.342 ./frontend/tailwind.config.ts:13:13
17:00:53.343 Module not found: Can't resolve 'tailwindcss-animate'
17:00:53.343   11 |   },
17:00:53.343   12 |
17:00:53.343 > 13 |   plugins: [require("tailwindcss-animate")],
17:00:53.344      |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
17:00:53.345   14 | };
17:00:53.345   15 |
17:00:53.345   16 | export default config;
17:00:53.346 
17:00:53.346 
17:00:53.346 
17:00:53.347 https://nextjs.org/docs/messages/module-not-found
17:00:53.347 
17:00:53.347 
17:00:53.373 ✓ Compiled successfully in 16.2s
17:00:53.385   Running TypeScript ...
17:01:01.837   Finished TypeScript in 8.5s ...
17:01:01.844   Collecting page data using 1 worker ...
17:01:02.514   Generating static pages using 1 worker (0/19) ...
17:01:02.825 Error occurred prerendering page "/forgot-password". Read more: https://nextjs.org/docs/messages/prerender-error
17:01:02.829 Error: supabaseKey is required.
17:01:02.830     at new cE (.next/server/chunks/ssr/frontend_app_lib_supabase_ts_0.he4em._.js:37:45742)
17:01:02.830     at module evaluation (.next/server/chunks/ssr/frontend_app_lib_supabase_ts_0.he4em._.js:37:50239)
17:01:02.831     at instantiateModule (.next/server/chunks/ssr/[turbopack]_runtime.js:853:9)
17:01:02.831     at getOrInstantiateModuleFromParent (.next/server/chunks/ssr/[turbopack]_runtime.js:877:12)
17:01:02.831     at Context.esmImport [as i] (.next/server/chunks/ssr/[turbopack]_runtime.js:281:20)
17:01:02.831     at module evaluation (.next/server/chunks/ssr/frontend_app_forgot-password_page_tsx_00gw7q.._.js:1:87)
17:01:02.831     at instantiateModule (.next/server/chunks/ssr/[turbopack]_runtime.js:853:9)
17:01:02.831     at getOrInstantiateModuleFromParent (.next/server/chunks/ssr/[turbopack]_runtime.js:877:12)
17:01:02.832     at Context.commonJsRequire [as r] (.next/server/chunks/ssr/[turbopack]_runtime.js:302:12) {
17:01:02.832   digest: '4016840855'
17:01:02.832 }
17:01:02.832 Export encountered an error on /forgot-password/page: /forgot-password, exiting the build.
17:01:02.857 ⨯ Next.js build worker exited with code: 1 and signal: null
17:01:02.902 npm error Lifecycle script `build` failed with error:
17:01:02.904 npm error code 1
17:01:02.904 npm error path /vercel/path0/frontend
17:01:02.905 npm error workspace frontend@0.1.0
17:01:02.905 npm error location /vercel/path0/frontend
17:01:02.906 npm error command failed
17:01:02.906 npm error command sh -c cross-env NODE_OPTIONS=--max-old-space-size=4096 next build
17:01:02.931 Error: Command "npm run build" exited with 1

--- a/frontend/app/lib/supabase.ts
+++ b/frontend/app/lib/supabase.ts
@@ -1,26 +1,18 @@
-import { createClient, SupabaseClient } from "@supabase/supabase-js";
+import { createClient } from "@supabase/supabase-js";
 
-const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL!;
-const supabaseKey = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY!;
+const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL;
+const supabaseKey = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY;
 
 export const isSupabaseConfigured =
   !!supabaseUrl && !!supabaseKey;
 
-declare global {
-  // eslint-disable-next-line no-var
-  var __supabase: SupabaseClient | undefined;
-}
-
 export const supabase =
-  globalThis.__supabase ??
-  createClient(supabaseUrl, supabaseKey, {
-    auth: {
-      persistSession: true,
-      autoRefreshToken: true,
-      detectSessionInUrl: true,
-    },
-  });
-
-if (typeof window !== "undefined") {
-  globalThis.__supabase = supabase;
-}
+  isSupabaseConfigured
+    ? createClient(supabaseUrl!, supabaseKey!, {
+        auth: {
+          persistSession: true,
+          autoRefreshToken: true,
+          detectSessionInUrl: true,
+        },
+      })
+    : null;

